modules/capacity/dmi/dmi.go

```diff
@@ -0,0 +1,341 @@
+package dmi
```

Comment:
You should have also moved the dmi_test.go file here to make sure we have this covered for potential bugs.